int64 num #Sets the maximum number of snapshots to be collected in the snapshot dir
---
string status #Sucess/Failure return message